py/desispec/io/fiberflat_vs_humidity.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+import fitsio
+import astropy.io.fits as fits
+from desiutil.log import get_logger
+from .meta import findfile
+from .util import native_endian
+
+def get_humidity(night,expid,camera) :
+    log=get_logger()
+    raw_filename=findfile("raw",night=night,expid=expid)
+    table=fitsio.read(raw_filename,"SPECTCONS")
+    keyword="{}HUMID".format(camera[0].upper())
+    unit=int(camera[1])
+    selection=(table["unit"]==unit)
+    if np.sum(selection)==0 :
+        log.warning("no unit '{}' in '{}'".format(unit,raw_filename))
+        return np.nan
+    humidity=float(table[keyword][selection][0])
```

Comment:
Consider checking for physical values here, and being robust to the column completely missing (same behavior as the camera missing from the table).

Reply:
done